package/batocera/core/batocera-configgen/configgen/configgen/generators/lindbergh/lindberghGenerator.py

```diff
@@ -79,6 +79,13 @@ def generate(self, system, rom, playersControllers, metadata, guns, wheels, game
         ### libraries
         self.setup_libraries(romDir, romName)
 
+        #some hardware expose /dev/dsp1|2 instead /dev/dsp required by 2spicy && gsevo
+        if not os.path.exists("/dev/dsp"):
```

Comment:
This should really be more robust. Not only should it check if it exists, but it should also be checking if it's pointing to the right place. I'm also not sure if this would affect other emulators or not.